Multiarch container  images push fails due to exec format error when pushing arm64 images from x86 host

Since moving to rules_oci, multiarch pushing of images fails with an exec format error.

```
INFO: Running command line: bazel-bin/push_push-example-hook-sidecar.sh --repository quay.io/bcarey/example-hook-sidecar --tag test-cross-builder-arm64
/root/.cache/bazel/_bazel_root/89960cf8ec9b8d55654b053cb7e89354/execroot/kubevirt/bazel-out/k8-fastbuild/bin/push_push-example-hook-sidecar.sh: line 52: ../jq_linux_arm64/jq: cannot execute binary file: Exec format error
make: *** [Makefile:40: bazel-push-images] Error 126
❌ ERROR: Build failed for architecture arm64
```

This is due to trying to run the arm64 binary of jq during the push. 

Similar issue open against rules_oci - https://github.com/bazel-contrib/rules_oci/issues/706

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -22,6 +22,13 @@ http_archive(
 )
 
 http_archive(
+    name = "package_metadata",
+    sha256 = "5bd0cc7594ea528fd28f98d82457f157827d48cc20e07bcfdbb56072f35c8f67",
+    strip_prefix = "supply-chain-0.0.6/metadata",
+    url = "https://github.com/bazel-contrib/supply-chain/releases/download/v0.0.6/supply-chain-v0.0.6.tar.gz",
+)
+
+http_archive(
     name = "rules_oci",
     sha256 = "e987cab7a35475cb9c9060fc3f338a1fc8896c240295a3272968b217acefd0cb",
     strip_prefix = "rules_oci-2.3.0",

